src/modules/home/HomePage.vue

```diff
@@ -4,7 +4,7 @@
       <font-awesome-icon class="transform -rotate-90" :icon="['fas', 'key']" />
     </template>
 
-    {{ t('homePage.passwordReset') }}
+    {{ `${t('homePage.passwordReset')}` }}
```

Comment:
I don't think this is needed?

Reply:
Mmmh, funny things are happening with the linter/fixer triggering upon committing. I managed to revert it – will let you mark this as resolved.

Reply:
Yeah we should definitely review whether the linter/fixer triggering on commit is useful or not, I find it a bit jarring